components/org.wso2.carbon.identity.api.server.workflow.management/org.wso2.carbon.identity.api.server.workflow.management.v1/src/main/java/org/wso2/carbon/identity/workflow/mgt/api/v1/util/Error.java

```diff
@@ -0,0 +1,146 @@
+//package org.wso2.carbon.identity.workflow.mgt.api.v1.util;
+//
+//import com.fasterxml.jackson.annotation.JsonProperty;
+//import com.hazelcast.org.codehaus.commons.nullanalysis.NotNull;
+//import io.swagger.annotations.ApiModelProperty;
+//
+//import java.util.Objects;
+//
+//import javax.validation.Valid;
+//
+//public class Error  {
+//
+//    private String code;
+//    private String message;
+//    private String description;
+//    private String traceId;
+//
+//    /**
+//     * An error code.
+//     **/
+//    public Error code(String code) {
```

Comment:
remove this class if not used

Reply:
fixed by : https://github.com/wso2/identity-api-server/pull/446/commits/3f0c88651577bd8b27c0f5d34ba6922881bcfe38